Sources/App/Controllers/KeywordController.swift

```diff
@@ -0,0 +1,45 @@
+import Fluent
+import Plot
+import Vapor
+
+
+struct KeywordController {
+
+    static func query(on database: Database, keyword: String) -> EventLoopFuture<[Package]> {
+        Package.query(on: database)
+            .with(\.$repositories)
+            .join(Repository.self, on: \Repository.$package.$id == \Package.$id)
+            .filter(
+                DatabaseQuery.Field.path(Repository.path(for: \.$keywords), schema: Repository.schema),
+                DatabaseQuery.Filter.Method.custom("@>"),
+                DatabaseQuery.Value.bind([keyword])
```

Comment:
This is not the query I originally intended to write. Fluent doesn't seem to support expressing `'foo' LIKE ANY("keywords")` so this is an equivalent workaround which performs similarly.

I've to a question open in the Vapor Discord but not takers yet: https://discord.com/channels/431917998102675485/684159753189982218/858402370144960552